src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -10791,7 +10846,7 @@ private ParameterListSyntax ParseLambdaParameterList()
 
                         // additional parameters
                         int tokenProgress = -1;
-                        while(IsMakingProgress(ref tokenProgress))
+                        while (IsMakingProgress(ref tokenProgress))
```

Comment:
This is code style change. Wouldn't it be more appropriate to move that to separate commit?